compiler/rustc_mir_dataflow/src/storage.rs

```diff
@@ -10,12 +10,9 @@ use rustc_middle::mir::{self, Local};
 pub fn always_live_locals(body: &mir::Body<'_>) -> BitSet<Local> {
     let mut always_live_locals = BitSet::new_filled(body.local_decls.len());
 
-    for block in body.basic_blocks() {
-        for statement in &block.statements {
-            use mir::StatementKind::{StorageDead, StorageLive};
-            if let StorageLive(l) | StorageDead(l) = statement.kind {
-                always_live_locals.remove(l);
-            }
+    for local in body.local_decls() {
+        if !local.always_storage_live {
+            always_live_locals.remove(local);
```

Comment:
Need to figure out how to get the `Local` from the `IndexVec`, not the `LocalDecl`. Might have to just iterate to identify the index...